specification/managednetwork/resource-manager/Microsoft.ManagedNetwork/preview/2019-06-01-preview/managedNetwork.json

```diff
@@ -1413,15 +1414,15 @@
       "description": "The resource model definition for a ARM tracked top level resource",
       "properties": {
         "tags": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "x-ms-mutability": [
-            "read",
-            "create",
-            "update"
-          ],
+            "type": "array",
```

Comment:
As this is a breaking change, is the APIs for 2019-06-01-preview already released?